### PR TITLE
Add some Windows-specific information for Running Code Locally

### PR DESCRIPTION
**Pull request by Mastermind497**
This includes how to get the header bits/stdc++ on windows, as well as the different compiler options available. This was made with the help of @wastingtime#1534

### CODE REVIEW

**Comment by thecodingwizard** [comment]
C++ With the Command Line also includes installation instructions for G++, etc. Should we move installation instructions into "Running Code Locally"? I think it fits better there anyway.

@cpinitiative/core-team

**Comment by bqi343** [comment]
sure

File: content/1_General/Running_Code_Locally.mdx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 id: running-code-locally
 title: Running Code Locally
-author: Benjamin Qi, Hankai Zhang, Anthony Wang, Nathan Wang, Nathan Chen, Owen Wang
+author: Benjamin Qi, Hankai Zhang, Anthony Wang, Nathan Wang, Nathan Chen, Owen Wang, Shourya Bansal, @wastingtime#1534
```
Comment (bqi343) [inline content/1_General/Running_Code_Locally.mdx:4]: `@wastingtime#1534`

real name?

**Review by bqi343 — CHANGES_REQUESTED** [review]

**Review by thecodingwizard — APPROVED** [review]
This is perfect. Thanks :heart:

**Pull request merged**